src/linux/wifi/core/AccessPointControllerLinux.cxx

```diff
@@ -119,6 +119,94 @@ HostapdHwModeToPropertyValue(Wpa::HostapdHwMode hwMode)
         throw AccessPointControllerException(std::format("Invalid hostapd hw_mode value {}", magic_enum::enum_name(hwMode)));
     }
 }
+
+std::string
+IeeeAkmSuiteToWpaKeyMgmtPropertyValue(Ieee80211AkmSuite akmSuite)
+{
+    switch (akmSuite) {
+    case Ieee80211AkmSuite::Reserved0:
+        return "";
+    case Ieee80211AkmSuite::Ieee8021x:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueWpaEap;
+    case Ieee80211AkmSuite::Psk:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueWpaPsk;
+    case Ieee80211AkmSuite::Ft8021x:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueFtEap;
+    case Ieee80211AkmSuite::FtPsk:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueFtPsk;
+    case Ieee80211AkmSuite::Ieee8021xSha256:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueWpaEapSha256;
+    case Ieee80211AkmSuite::PskSha256:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueWpaPskSha256;
+    case Ieee80211AkmSuite::Tdls:
+        return "";
+    case Ieee80211AkmSuite::Sae:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueSae;
+    case Ieee80211AkmSuite::FtSae:
+        return Wpa::ProtocolHostapd::PropertyWpaKeyMgmtValueFtSae;
+    case Ieee80211AkmSuite::ApPeerKey:
+        return "";
+    case Ieee80211AkmSuite::Ieee8021xSuiteB:
+        return "";
```

Comment:
Need to add one for this:

```suggestion
    // Next line in ProtocolHostapd.hxx:
    static constexpr auto PropertyWpaKeyMgmtValueWpaEapSuiteB = "WPA-EAP-SUITE-B";

    case Ieee80211AkmSuite::Ieee8021xSuiteB:
        return PropertyWpaKeyMgmtValueWpaEapSuiteB;
```

Reply:
For some reason, I don't see that in the hostapd.conf file that I have, it only has SuiteB192, but perhaps it was just forgotten in the comments for that example conf file

Reply:
Yeah that appears to be the case, since I see both suite B and suite B192 in `config_file.c`